frappe/public/js/frappe/form/form.js

```diff
@@ -1658,6 +1658,21 @@ frappe.ui.form.Form = class FrappeForm {
 		frappe.route.on('change', () => driver.reset());
 		driver.start();
 	}
+
+	set_fields_as_options(fieldname, doctype, condition, default_options) {
```

Comment:
```suggestion
	set_fields_as_options(fieldname, ref_doctype, filter, default_options=[]) {
```
Can you also add a comment to describe what the function does? It might not be clear for users with no context.